src/patternfly/components/Select/select.scss

```diff
@@ -0,0 +1,325 @@
+  --pf-c-select__menu--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-select__menu--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-select__menu--BorderColor: var(--pf-global--BorderColor);
+  --pf-c-select__menu--BoxShadow: var(--pf-global--BoxShadow--md);
+  --pf-c-select__menu--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-select__menu--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-select__menu--Top: calc(100% + var(--pf-global--spacer--xs)); // The top of the menu must be pushed down to create space between the toggle and menu
+  --pf-c-select__menu--ZIndex: var(--pf-global--ZIndex--xs);
+  --pf-c-select__menu--m-top--Transform: translateY(calc(-100% - var(--pf-global--spacer--xs))); // The "dropup" menu must be transformed up and this calculates how much to create space between the toggle and menu
+
+  // Menu Item
+  --pf-c-select__menu-item--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-select__menu-item--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-select__menu-item--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-select__menu-item--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-select__menu-item--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-select__menu-item--FontWeight: var(--pf-global--FontWeight--normal);
+  --pf-c-select__menu-item--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-select__menu-item--Color: var(--pf-global--Color--dark-100);
+  --pf-c-select__menu-item--disabled--Color: var(--pf-global--Color--dark-200);
+  --pf-c-select__menu-item--hover--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+
+  // Match within a menu item when using typeahead
+  --pf-c-select__menu-item--match--FontWeight: var(--pf-global--FontWeight--bold);
+
+  // Menu Item Separator
+  --pf-c-select__separator--Height: var(--pf-global--BorderWidth--sm);
+  --pf-c-select__separator--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+
+  // Menu group
+  // add a little space if it's not the first group because the first one also has the menu padding
+  --pf-c-select__menu-group--not-first--PaddingTop: var(--pf-global--spacer--sm);
+
+  // Menu section heading
+  // Paddings are set to always line up with the other menu items
+  --pf-c-select__menu-group-title--PaddingTop: var(--pf-c-select__menu-item--PaddingTop);
+  --pf-c-select__menu-group-title--PaddingRight: var(--pf-c-select__menu-item--PaddingRight);
+  --pf-c-select__menu-group-title--PaddingBottom: var(--pf-c-select__menu-item--PaddingBottom);
+  --pf-c-select__menu-group-title--PaddingLeft: var(--pf-c-select__menu-item--PaddingLeft);
+  --pf-c-select__menu-group-title--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-select__menu-group-title--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-select__menu-group-title--Color: var(--pf-global--Color--dark-200);
+
+  position: relative;
+  display: inline-block;
+  max-width: 100%;
+}
+
+.pf-c-select__toggle {
```

Comment:
I was following dropdown. MDN says "The [pointer] cursor is a pointer that indicates a link. Typically an image of a pointing hand." I wouldn't think from this that it would be a pointer cursor. But I can do that if you want.